### PR TITLE
Extra credit

### PR DESCRIPTION
**Pull request by bantuist**
- I added  get_appointments and update_submission methods
- I stubbed the VCR specs, but I'm still figuring out how to configure VCR with WebMock

```
 WebMock::NetConnectNotAllowedError:
       Real HTTP connections are disabled. Unregistered request: POST https://www.bloc.io/api/v1/sessions with body 'email=REMOVED&password=REMOVED' with headers {'Accept'=>'*/*', 'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'User-Agent'=>'Ruby'}
     
       You can stub this request with the following snippet:
     
       stub_request(:post, "https://www.bloc.io/api/v1/sessions").
         with(:body => "email=REMOVED&password=REMOVED",
              :headers => {'Accept'=>'*/*', 'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'User-Agent'=>'Ruby'}).
         to_return(:status => 200, :body => "", :headers => {})
```

Also, was I only using the json gem for parsing? Is it alright to remove it since I'm using HTTParty parsed_response?

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -2,23 +2,61 @@ PATH
...
+    addressable (2.5.0)
+      public_suffix (~> 2.0, >= 2.0.2)
     awesome_print (1.7.0)
+    coderay (1.1.1)
+    crack (0.4.3)
+      safe_yaml (~> 1.0.0)
+    diff-lcs (1.2.5)
+    hashdiff (0.3.0)
     httparty (0.14.0)
       multi_xml (>= 0.5.2)
```
Comment (bantuist) [inline Gemfile.lock]: Wouldn't json be listed here with multi_xml if it was a httparty dependency?

File: lib/kele.rb [hunk 2]
```diff
@@ -12,24 +14,45 @@ class Kele
...
+    my_appointments.each.with_index(1) do |appointment, index|
+      starts_at = DateTime.parse(appointment["starts_at"]).strftime("%a %m/%d/%y %l:%M%p")
+      ends_at = DateTime.parse(appointment["ends_at"]).strftime("%l:%M%p")
+      puts "#{index}. #{starts_at}\u{2013}#{ends_at}"
+    end
+  end
+
+  private
+
+  def get_auth_token(email, password)
```
Comment (bantuist) [inline lib/kele.rb]: I made this private again because I'm not using it in the spec now.

File: lib/spec/kele_spec.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+RSpec.describe Kele, type: :request do
+  let(:student) { Kele.new(ENV['STUDENT_EMAIL'], ENV['STUDENT_PASSWORD']) }
+
+  # describe '#get_auth_token' do
+  #   it 'returns a String', vcr: { cassette_name: :get_auth_token } do
+  #     expect(student.get_auth_token(ENV['STUDENT_EMAIL'], ENV['STUDENT_PASSWORD'])).to be_a(String)
+  #   end
+  # end
+
+  # Object#instance_variable_get: returns the value of the given instance variable since initialize returns the instance itself.
```
Comment (bantuist) [inline lib/spec/kele_spec.rb]: I found this method to get the return value instead of the instance.

**Comment by bantuist** [comment]
- Test responses
  - #initialize
  - #get_me
  - #get_mentor_availability
  - #get_appointments
  - #get_roadmap
  - #get_checkpoint
- Add dotenv and setup ENV variables for spec
- Setup webmock and vcr in spec_helper to record cassettes
- Update README

**Comment by johnksawers** [comment]
Looks great, ready to merge

**Pull request merged**